src/components/image-tooltip/ImageTooltipContent.tsx

```diff
@@ -3,35 +3,34 @@ import classNames from 'classnames';
 
 import './ImageTooltipContent.scss';
 
+type OnImageLoad = () => void;
+
 export type ImageTooltipContentProps = {
     /** A React component representing the image used in the visual tooltip */
     children: React.ReactElement;
     /** A string to be used in the tooltip's paragraph content */
     content: string;
+    /** A callback triggered onLoad for the image element */
+    onImageLoad: OnImageLoad;
     /** A string to be used in the tooltip's title heading */
     title: string;
 };
 
-function appendTooltipImageChildClass(child: React.ReactElement): React.ReactElement | null {
-    if (child) {
-        const {
-            props: { className: existingClasses },
-        } = child;
-        const className = classNames(existingClasses, 'bdl-ImageTooltipContent-imageChild');
-
-        return React.cloneElement(child, {
-            className,
-        });
-    }
+function cloneTooltipChildWithNewProps(child: React.ReactElement, onImageLoad: OnImageLoad): React.ReactElement {
+    const {
+        props: { className: existingClasses },
+    } = child;
+    const className = classNames(existingClasses, 'bdl-ImageTooltipContent-imageChild');
 
-    return null;
+    return React.cloneElement(child, {
+        className,
+        onLoad: onImageLoad,
+    });
 }
 
-const ImageTooltipContent = ({ children, content, title }: ImageTooltipContentProps) => (
+const ImageTooltipContent = ({ children, content, onImageLoad, title }: ImageTooltipContentProps) => (
     <div className="bdl-ImageTooltipContent">
-        <div className="bdl-ImageTooltipContent-image">
-            {React.Children.map(children, child => appendTooltipImageChildClass(child))}
```

Comment:
We can replace this `.map()` with just the call to our cloning function since our parent does a `React.Children.only` check.